daprdocs/content/en/operations/configuration/api-allowlist.md

```diff
@@ -59,9 +65,47 @@ metadata:
 spec:
   api:
     allowed:
-    - name: state
-      version: v1
-      protocol: grpc
+      - name: state
+        version: v1
+        protocol: grpc
+```
+
+### Using a denylist
+
+#### Disabling specific HTTP APIs
+
+The following example disables the state `v1.0` HTTP API, allowing all other HTTP APIs:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: myappconfig
+  namespace: default
+spec:
+  api:
+    denied:
+      - name: state
+        version: v1.0
+        protocol: http
+```
+
+#### Disabling specific gRPC APIs
+
+The following example disables the state `v1` gRPC API, allowing all other gRPC APIs:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: myappconfig
+  namespace: default
+spec:
+  api:
+    denied:
+      - name: state
+        version: v1
+        protocol: grpc
 ```
 
 ### List of Dapr APIs
```

Comment:
Just seen that this list does not include "Configuation" which is now V1 and should be enabled.

Also I presume that **all** alpha API can, like workflow, Lock and Crypto can also be blocked. If they can they provide an example of what is needed for the version. If not then, we should understand why and put a comment here saying the reason for not blocking them, since this may be a security concern.

Reply:
Yes, the list below should probably be revised. Let's do that in a separate PR.

Yup, all APIs can be blocked, even alpha ones.

Reply:
Why not add to this PR?

Reply:
@ItalyPaleAle - Need you to confirm the API values to ensure this list is correct. Can you please take look?

Reply:
Please take a look now